avocado_vt/vt_agent/src/avocado_vt/agent/core/data_dir.py

```
@@ -0,0 +1,145 @@
+    :type public: bool
+    :return: The path to the created temporary directory.
+    :rtype: str
+    """
+    data_dir_path = get_data_dir()
+    if not os.path.exists(data_dir_path):
+        try:
+            os.makedirs(data_dir_path)
+        except OSError as e:
+            LOG.error(
+                "Failed to create data directory %s for temp dir: %s", data_dir_path, e
+            )
```

Ruff (0.13.3)

Use `logging.exception` instead of `logging.error`

Replace with `exception`

(TRY400)